envs/strelka/strelka-2.9.10.yaml

```diff
@@ -0,0 +1,28 @@
+
+name: strelka-2.9.10
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1
+  - _openmp_mutex=4.5
+  - ca-certificates=2019.11.28
+  - certifi=2019.11.28
+  - libffi=3.2.1
+  - libgcc-ng=9.2.0
+  - libgomp=9.2.0
+  - libstdcxx-ng=9.2.0
+  - manta=1.6.0
```

Comment:
Is Manta a dependency of Strelka? Or was this added by you? If so, could you create a Strelka conda environment without this extraneous dependency since we already have a manta env? Thanks!

Reply:
I exported it from a pre-existing environment but ya I can remove it.